MetaMask/metamask-extension
# UX: Multichain: Address Copy Button

## Explanation

Provides the new copy button component for use around the extension.

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/375

## Screenshots/Screencaps

<img width="379" alt="CopyButton" src="https://user-images.githubusercontent.com/46655/225110536-4938cc28-7250-4667-bed3-847219fb1c2d.png">


## Manual Testing Steps

1.  Click button
2. See icon change

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
